axis.go

```diff
@@ -384,7 +384,7 @@ func (DefaultTicks) Ticks(min, max float64) (ticks []Tick) {
 	}
 	majorDelta := float64(majorMult) * tens
 	val := math.Floor(min/majorDelta) * majorDelta
-	prec := maxInt(precisionOf(min), precisionOf(max))
+	prec := precisionOf(majorDelta)
```

Comment:
Yes, thanks to @jasonpfox for that observation. I don't really know what I was thinking when I wrote the original.